train/README.md

```diff
@@ -34,6 +34,33 @@ torchrun --nproc_per_node=2 --master_port=20001 train.py \
     --model_max_length 4096 \
     --gradient_checkpointing True \
     --output_dir functionary-v1
+
+# 2xA100 80GB WITH evaluation
+torchrun --nproc_per_node=2 --master_port=20001 train.py \
```

Comment:
Can we remove the "WITHOUT Evaluation" bash code and only keep the "WITH" eval one for simplicity